shell/aoe_translate.php

```diff
@@ -50,6 +56,57 @@ public function generateStoreTranslationsActionHelp()
     }
 
     /**
+     * @return void
+     */
+    public function exportTranslationTableAction()
+    {
+        $resourceCollection = Mage::getResourceModel('aoe_translate/log_collection');
+        foreach ($resourceCollection as $item) {
+            $fileHandle = $this->_getExportHandleForLocale($item->getLocale());
```

Comment:
Perhaps pre-process the collection to group entries into sub-arrays by locale. Then you can create, add, then close each file in turn.